addons/pkg-npm/jaxrs/src/main/java/org/commonjava/indy/pkg/npm/jaxrs/NPMContentAccessResource.java

```diff
@@ -145,13 +159,27 @@ public Response doHead(
     public Response doHead(
                     final @ApiParam( allowableValues = "hosted,group,remote", required = true ) @PathParam( "type" ) String type,
                     final @ApiParam( required = true ) @PathParam( "name" ) String name,
-                    final @PathParam( "packageName" ) String packageName, final @PathParam( "version" ) String version,
+                    final @PathParam( "packageName" ) String packageName,
+                    final @ApiParam( allowableValues = CROSS ) @PathParam( "version" ) String version,
```

Comment:
@yma88 I was just wondering if you could take out the `allowableValues = CROSS` in @ApiParam here. I think the method split is good still.

(This is a minor point, of course.)